index.html

```diff
@@ -1786,8 +1786,10 @@ <h3>Processing capabilities</h3>
       <var>value</var>.
 
      <dt><var>name</var> is the key of an <a>extension capability</a>
-     <dd><p>Let <var>deserialized</var> be the result of <a>trying</a>
+     <dd><p>If <var>name</var> is known to the implementation,
```

Comment:
Perhaps “known to the implementation” is a bit weak?

What we’re trying to say here is that if the capability is one of the defined, standard capabilities in the specification _or_ a known extension capability, it should be accepted.

Or more to the point, if the extension capability is _unknown_ it must be rejected.

Reply:
Since this sentence is under the heading "extension capability", My intention is "name is known" would mean a known extension capability.

For unknown extension capability, I recall the conclusion at TPAC was to accept them, not reject them. This is stated in the resolution at https://www.w3.org/2019/09/19-webdriver-minutes.html#resolution04.

Reply:
Sorry for the delay here…

That makes sense to me.